active/community.md

```diff
@@ -0,0 +1,86 @@
+We can support them through training and taking action to make their lives easier.
+
+The broad goals of the group are:
+  - facilitate and centralise discussion between moderators and admins on the various platforms
+  - coordinate efforts for training and processes around moderation
+  - provide support to improve the platforms to lighten moderation and admin workloads
+  - provide training and support to the wider community on how to use the platforms effectively
+  - ensure consistency that our online Django communities are welcoming and safe spaces
+
+- What actions are you proposing the WG be allowed to take directly?
+  - Creation and management of processes related to community platforms.
+    - Adding/removing moderators & other roles
+    - Training for moderators
+    - Ensuring moderation is consistent across teams and platforms
+  - Taking ideas from the community to improve the community spaces.
+- Which actions will the WG take back to the Board for votes?
+  - Moving platforms if required
+  - Budget to improve existing platforms
+
+There is a lot of potential ideas in this space. Part of the WG will be setting up a process to take ideas and have a backlog of these ideas.
+
+### Delegated responsibilities
+
+With regards to Django Software Foundation responsibilities and resources, the group operates with:
+
+- Admin privileges to members of the forum & Discord or existing admin representive joining as a member.
+
+## Initial membership
+
+- Chair: TBD
+- Co-Chair: TBD
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBD
+- Other members:
+  - Andrew Miller
+
+## Future membership
+
+Answer the following questions here:
+
+- Who is eligible to join? Any volunteer, or are there specific requirements?
+  - Anyone can join, but they at least need to be known within the community.
+- How do people who want to join sign up / volunteer / express interest?
+  - Send one of the WG members a message on Discord or the Forum
+- How will decisions on adding/removing members be handled?
+  - Direct membership: new members may self-nominate; members will serve for a year minimum with the option to step out after each year. New members will get approved by existing members of the WG.
+
+
+## Budget
+
+- How much money or spending discretion do you need?
+  - None to begin with, however I would expect over time budget to get allocated for supporting the Forum and Discord  server. See below for [ideas](#budget-ideas)
+- How do you want that money allocated:
+  - None to begin with.
+
+## Comms
+
+Where will discussions and activities take place?
+
+Suggestions:
+
+- Private Channel in the Discord Server and private category in the forum, possibly also a channel is the DSF slack.
+- This space should be shared with the CoC Commitee or another space for shared comms across platforms.
```

Comment:
Please make sure to share this WG proposal with them if you haven’t already

Reply:
Will do!